ifrs17/Import/ImportStorage.ipynb

```diff
@@ -477,6 +479,15 @@
                 "\n    public IEnumerable<string> GetClaims() => hierarchyCache.Get<AmountType>(AmountTypes.CL).Descendants().Select(x => x.SystemName);",
                 "\n    public IEnumerable<string> GetCdr() => hierarchyCache.Get<AmountType>(AmountTypes.CDR).Descendants(includeSelf: true).Select(x => x.SystemName);",
                 "\n    public IEnumerable<string> GetCoverageUnits() => hierarchyCache.Get<AmountType>(AmountTypes.CU).Descendants(includeSelf: true).Select(x => x.SystemName);",
+                "\n",
+                "\n    private YieldCurve LogNoYcError(string dn, bool locked) =>",
+                "\n        DataNodeDataBySystemName.TryGetValue(dn, out var dataNodeData) ?  ",
```

Comment:
You don't need to validate DataNodeDataBySystemName.TryGetValue(dn:
if you get to here, the DataNode has alredy been validated (parser).

Reply:
done